File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.java
import java.util.*;

class Solution {
    public String solution(String my_string, int[] indices) {
        String answer = "";
        int idx = 0;
        Arrays.sort(indices);
        for(int i=0;i<my_string.length();i++){
            
            if(idx != indices.length && indices[idx] == i){
                idx++;
                continue;
            }
            answer += my_string.charAt(i);
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.py
def solution(my_string, indices):
    answer = ''
    indices.sort()
    idx = 0
    for i in range(0, len(my_string)):
        if len(indices) != idx and indices[idx] == i:
            idx += 1
            continue
        answer += my_string[i]
    return answer